packages/engine/lib/stateful/src/field/mod.rs

```diff
@@ -1,13 +1,23 @@
-//! Provides an hEngine-specific representation of Arrows `Field` and `DataType`.
+//! Provides an hEngine-specific representation of Arrows `Field`, `DataType`, and `Schema`.
 //!
-//! As only a selection of [`arrow::DataType`]s is provided to hEngine, this module defines
-//! [`FieldType`], which uses Arrow types underneath and adds an `Any` type, which internally uses
-//! a JSON encoded `String` type. As [`FieldType`] can't be used in [`arrow::Field`], this module
-//! provides [`FieldSpec`] to associate a name with the [`FieldType`].
+//! As only a selection of Arrow [`DataType`]s is supported at hEngine, this module defines
```

Comment:
Similar comment as above, we don't expose Arrow DataTypes to users, we expose our own datatypes, which happen to be implemented **using** Arrow.

Reply:
Addressed this in b1b7b08f4269775b94e6a768bb1db2015ff6030e